src/client/app/components/HeaderComponent.tsx

```diff
@@ -3,36 +3,24 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import * as React from 'react';
-import { Link } from 'react-router-dom';
+import { Link, useLocation } from 'react-router-dom';
+import { selectOptionsVisibility } from '../redux/slices/appStateSlice';
+import { useAppSelector } from '../redux/reduxHooks';
+import HeaderButtonsComponent from './HeaderButtonsComponent';
 import LogoComponent from './LogoComponent';
 import MenuModalComponent from './MenuModalComponent';
-import HeaderButtonsComponent from './HeaderButtonsComponent';
-import { useSelector } from 'react-redux';
-import { State } from '../types/redux/state';
-import getPage from '../utils/getPage';
 
 /**
  * React component that controls the header strip at the top of all pages
  * @returns header element
  */
 export default function HeaderComponent() {
-	const siteTitle = useSelector((state: State) => state.admin.displayTitle);
-	const showOptions = useSelector((state: State) => state.graph.optionsVisibility);
-
-	const divStyle = {
-		marginTop: '5px',
-		paddingBottom: '5px'
-	};
-	const largeTitleStyle = {
-		display: 'inline-block'
-	};
-	const smallTitleStyle = {
-		display: 'inline-block',
-		marginTop: '10px'
-	};
+	const siteTitle = useAppSelector(state => state.admin.displayTitle);
```

Comment:
Another palce where admin state is directly accessed.

Reply:
Corrected.